view/js/dropzone-factory.js

```diff
@@ -26,10 +26,10 @@ var DzFactory = function (max_imagesize) {
 					const targetTextarea = document.getElementById(textareaElementId);
 					if (targetTextarea.setRangeText) {
 						//if setRangeText function is supported by current browser
-						targetTextarea.setRangeText(' ' + $.trim(serverResponse) + ' ');
+						targetTextarea.setRangeText(' ' + serverResponse + ' ');
```

Comment:
No need for the wrapping white space anymore:
```suggestion
						targetTextarea.setRangeText(serverResponse);
```